src/FurCoNZ.Web/Areas/Admin/Views/Orders/Index.cshtml

```diff
@@ -53,6 +56,9 @@
             <td>
                 @item.AmountOwing.ToString("C")
             </td>
+                <td>
+                    @Html.DisplayFor(modelItem => item.Status)
```

Comment:
This lambda doesn't seem to use `modelItem`, is this correct?

Reply:
Because there's no `DisplayFor(object ...)` just `DisplayFor(expression ...)` So `modelItem` is a throwaway. yes it's a hack, but it works